src/main/java/hudson/plugins/accurev/AccurevLauncher.java

```diff
@@ -484,17 +484,6 @@ private static ProcStarter createProcess(
     ProcStarter starter = launcher.launch().cmds(machineReadableCommand);
     Node n = workspaceToNode(directoryToRunCommandFrom);
     environmentVariables.putAll(buildEnvironment(n, listener));
-    String path = null;
-    FilePath filePath = null;
-    if (null != n) {
-      filePath = n.getRootPath();
-    }
-    if (null != filePath) {
-      path = filePath.getRemote();
-    }
-    if (StringUtils.isNotBlank(path)) {
-      environmentVariables.putIfAbsent("ACCUREV_HOME", path);
-    }
```

Comment:
Why is this removed?

It's unrelated to ACCUREV_SUBPATH and tries to fix an issue when running accurev from Jenkins.

Reply:
Could you please remove it from this PR and file another?